test/e2e/interest-groups/index.test.js

```diff
@@ -1,52 +1,24 @@
 /* eslint-disable no-undef */
 
-describe('Initial test', () => {
-	beforeAll(async () => {
-		await page.goto('http://localhost:3000/test/e2e/interest-groups/');
-	});
-
-	it('should be titled "interest-groups.test"', async () => {
-		await expect(await page.title()).toMatch('interest-groups.test');
-	});
-
-	it('should provide the fledge code', async () => {
-		await expect(await page.title()).toMatch('interest-groups.test');
-		const fledge = await page.evaluate(() => window.fledge.fledge);
-		expect(Object.keys(fledge)).toEqual([ 'joinAdInterestGroup', 'leaveAdInterestGroup' ]);
-	});
-});
-
```

Comment:
this was moved into a new file to test the API itself on the `window`

Reply:
you seemed to have forgotten to include the new file in this PR.

Reply:
yeah, its in the `auction` branch which will eventually be merged in either here or `main`.